cartography/models/duo/user.py

```diff
@@ -81,7 +81,7 @@ class DuoGroupToDuoUserRel(CartographyRelSchema):
         {'group_id': PropertyRef('group_id')},
     )
     direction: LinkDirection = LinkDirection.INWARD
```

Comment:
We might want to point this the other direction. For example with MEMBER_OF_OKTA_GROUP we do `(:OktaUser)-[:MEMBER_OF_OKTA_GROUP]->(:OktaGroup)`. Right now this is pointing to the DuoUser and not the DuoGroup.

Reply:
changed